packages/graph/hash_graph/lib/graph/src/api/rest/utoipa_typedef/mod.rs

```diff
@@ -1 +1,41 @@
+use serde::{Deserialize, Serialize};
+use utoipa::{
+    openapi::{KnownFormat, Object, ObjectBuilder, Schema, SchemaFormat, SchemaType},
+    ToSchema,
+};
+
+use crate::identifier::{knowledge::EntityId, TransactionTimestamp};
+
 pub mod subgraph;
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct EntityIdAndTimestamp {
+    pub base_id: EntityId,
+    pub timestamp: TransactionTimestamp,
+}
+
+// WARNING: This MUST be kept up to date with the struct names and serde attributes
+//   Necessary because Timestamp doesn't implement ToSchema
+impl ToSchema for EntityIdAndTimestamp {
+    fn schema() -> Schema {
+        ObjectBuilder::new()
+            .property(
+                "baseId",
+                // Apparently OpenAPI doesn't support const values, the best you can do is
+                // an enum with one option
```

Comment:
Appreciate this might have been copied but I don't think this makes sense here?

Reply:
I worked around this: fadffa6a6e351a054ecea93d4e989e74dd785a73